lib/kitchen/unit_element.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+module Kitchen
+  # An element for a unit
+  #
+  class UnitElement < ElementBase
+
+    # Creates a new +UnitElement+
+    #
+    # @param node [Nokogiri::XML::Node] the node this element wraps
+    # @param document [Document] this element's document
+    #
+    def initialize(node:, document: nil)
+      super(node: node,
+            document: document,
+            enumerator_class: UnitElementEnumerator,
+            short_type: :unit)
+    end
+
+    def title
```

Comment:
Add method documentation here (the YARD docs like above and below)

Reply:
Needs 

```
# @return [Element]
```

Reply:
🙈